modules/channel-web/src/views/lite/components/messages/Message.tsx

```diff
@@ -175,7 +175,23 @@ class Message extends Component<MessageProps> {
       return '* Cannot display message *'
     }
 
-    const type = this.props.type || (this.props.payload && this.props.payload.type)
+    let type = this.props.type || (this.props.payload && this.props.payload.type)
+
+    if (type === 'say_something') {
+      let text = (this.props.payload.text || this.props.payload.preview).slice(2)
```

Comment:
Can you also ensure that the standard "say something" behavior (To send a text string in the chat) also works here? This line makes me wonder if it's going to work